src/twisted/python/test/test_url.py

```diff
@@ -793,6 +799,25 @@ def test_technicallyTextIsIterableBut(self):
             URL(path=u'foo')
         self.assertEqual(
             str(raised.exception),
-            "expected iterable of text for path, got text itself: {}"
+            "expected iterable of text for path, not: {}"
```

Comment:
I am ok with this exception... no need to be that strict on repr... at least for now :)